pinecone/deprecated/legacy_utils.py

```diff
@@ -0,0 +1,88 @@
+import re
```

Comment:
Leftovers, hopefully to be deleted soon pending investigation on vector_column_service.proto